tests/evaluator_tests.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+import numpy
+import unittest
+from lib.abstract_recommender import AbstractRecommender
+from lib.collaborative_filtering import CollaborativeFiltering
+from lib.evaluator import Evaluator
+from util.data_parser import DataParser
+from util.model_initializer import ModelInitializer
+
+
+class TestcaseBase(unittest.TestCase):
+    def setUp(self):
+        """
+        Setup method that is called at the beginning of each test.
+        """
+        self.documents, self.users = 8, 10
+        documents_cnt, users_cnt = self.documents, self.users
+        self.config = {'n_factors': 5, '_lambda': 0.01}
+        self.n_iterations = 15
+        self.initializer = ModelInitializer(self.config.copy(), self.n_iterations)
+
+        self.n_recommendations = 1
+
+        def mock_get_ratings_matrix(self=None):
+
+            return [[int(not bool((article + user) % 3)) for article in range(documents_cnt)]
+                    for user in range(users_cnt)]
+        self.ratings_matrix = numpy.array(mock_get_ratings_matrix())
+        setattr(DataParser, "get_ratings_matrix", mock_get_ratings_matrix)
+
+        evaluator = Evaluator(self.ratings_matrix)
+        config = {'n_factors': 5, '_lambda': 0.01}
```

Comment:
I meant the config variable, it is never used. I ran the tests without them and they work fine. you have two config variables, one is instance variable and the other is this one. this one is never used.